docs/PremiumApps.md

```diff
@@ -0,0 +1,488 @@
+
+Assumption: the ES app tarball exists in an s3 bucket folder named "es_app" under the parent folder "security-team-apps" in your s3 bucket.
+
+```yaml
+apiVersion: enterprise.splunk.com/v4
+kind: Standalone
+metadata:
+  name: example
+  finalizers:
+  - enterprise.splunk.com/delete-pvc
+spec:
+  replicas: 1
+  appRepo:
+    appsRepoPollIntervalSeconds: 60
+    defaults:
+      volumeName: volume_app_repo
+      scope: local
+    appSources:
+      - name: esApp
+        location: es_app/
+        scope: premiumApps
+        premiumAppsProps:
+          type: enterpriseSecurity
+          esDefaults:
+             sslEnablement: ignore
+    volumes:
+      - name: volume_app_repo
+        storageType: s3
+        provider: aws
+        path: security-team-apps/
+        endpoint: https://s3-us-west-2.amazonaws.com
+        region: us-west-2
+        secretRef: splunk-s3-secret
+```
+
+#### Install ES on a standalone splunk deployment with sslEnablement value strict
+
+Here is an example of a standalone CR with the `sslEnablement=strict`. 
+
+Assumption: the ES app tarball exists in an s3 bucket folder named "es_app" under the parent folder "security-team-apps" in your s3 bucket.
+
+```yaml
+apiVersion: enterprise.splunk.com/v4
+kind: Standalone
+metadata:
+  name: example
+  finalizers:
+  - enterprise.splunk.com/delete-pvc
+spec:
+  extraEnv:
+    - name: SPLUNK_HTTP_ENABLESSL
+      value : "true"
+  replicas: 1
+  appRepo:
+    appsRepoPollIntervalSeconds: 60
+    defaults:
+      volumeName: volume_app_repo
+      scope: local
+    appSources:
+      - name: esApp
+        location: es_app/
+        scope: premiumApps
+        premiumAppsProps:
+          type: enterpriseSecurity
+          esDefaults:
+             sslEnablement: ignore
+    volumes:
+      - name: volume_app_repo
+        storageType: s3
+        provider: aws
+        path: security-team-apps/
+        endpoint: https://s3-us-west-2.amazonaws.com
+        region: us-west-2
+        secretRef: splunk-s3-secret
+```
+Following YAML file will also use sslEnablement as strict given you have not provided this parameter.
+
+```yaml
+apiVersion: enterprise.splunk.com/v4
+kind: Standalone
+metadata:
+  name: example
+  finalizers:
+  - enterprise.splunk.com/delete-pvc
+spec:
+  extraEnv:
+    - name: SPLUNK_HTTP_ENABLESSL
+      value : "true"
+  replicas: 1
+  appRepo:
+    appsRepoPollIntervalSeconds: 60
+    defaults:
+      volumeName: volume_app_repo
+      scope: local
+    appSources:
+      - name: esApp
+        location: es_app/
+        scope: premiumApps
+        premiumAppsProps:
+          type: enterpriseSecurity
+    volumes:
+      - name: volume_app_repo
+        storageType: s3
+        provider: aws
+        path: security-team-apps/
+        endpoint: https://s3-us-west-2.amazonaws.com
+        region: us-west-2
+        secretRef: splunk-s3-secret
+```
+
+#### Install ES on a standalone splunk deployment with sslEnablement value auto
```

Comment:
Do we need a separate section for different sslEnablement option? We can indicate in a single section to change sslEnablement option. We can have one example with SPLUNK_HTTP_ENABLESSL to show them they can enable ssl on splunkd with this.

Reply:
good point Arjun, this section was updated yesterday.